config/environment.py

```diff
@@ -65,7 +65,7 @@
     OAUTH_TOKEN = os.environ.get('OAUTH_TOKEN')
     SECRET_KEY = os.environ.get('SECRET_KEY') or 'NO SECRETS'
     HOOK_SECRET_KEY = os.environ.get('HOOK_SECRET_KEY') or 'SECRETS'
-    ANON_REPORTING_ENABLED = os.environ.get('ANON_REPORTING_ENABLED') or False
+    ANONYMOUS_REPORTING = os.environ.get('ANONYMOUS_REPORTING') or False
```

Comment:
I'm late since it's been merged already, but this should probably be `or 'OFF'`

Reply:
Oh... you're right. :)

I'll send in a PR, because it was my bad for merging too early. 🙈